client/app/components/login-form.js

```diff
@@ -0,0 +1,29 @@
+import Ember from 'ember';
+
+const { service } = Ember.inject;
+
+export default Ember.Component.extend({
+  session: service('session'),
```

Comment:
Yeah I think wrapping it in our own session service would be better. Then probably inject it across all controllers and routes maybe. 

- Sean Copenhaver

> On Jan 11, 2016, at 9:54 AM, Matthew Philyaw <notifications@github.com> wrote:
> 
> In client/app/components/login-form.js:
> 
> > @@ -0,0 +1,29 @@
> > +import Ember from 'ember';
> > +
> > +const { service } = Ember.inject;
> > +
> > +export default Ember.Component.extend({
> > +  session: service('session'),
> So the examples I have see use this pattern, but I also noticed this session-account in the dummy app they provide for ember-simple-auth.
> 
> which is a service that wraps up access to it. Maybe doing something like this and injecting at the app level would be a good idea?
> 
> They use it in other components to load the user from session, but I'm curious if we could not attach a login/register function to a service like that and have it hide the ember-simple-auth session and of course injecting it in like you were saying at the app level.
> 
> The it would read like this.get('session-something').login(), this.get('session-something').register() and what ever else we add.
> 
> —
> Reply to this email directly or view it on GitHub.
>